assembl/static2/js/app/pages/voteSession.jsx

```diff
@@ -56,23 +56,24 @@ export type Proposal = {|
 |};
 
 type Props = {
-  loading: boolean,
-  title: string,
-  subTitle: string,
-  seeCurrentVotes: boolean,
+  addGaugeVote: Function,
+  addTokenVote: Function,
```

Comment:
Same here

Reply:
this is not related with the pr, this is just a move for alphabetic order